src/test/ui/async-await/issue-70935-complex-spans.rs

```diff
@@ -1,5 +1,6 @@
 // edition:2018
 // revisions: normal drop_tracking
+// [normal]compile-flags:-Zdrop-tracking=no
```

Comment:
nit: this name will be wrong once it's on by default
```suggestion
// [no-drop-tracking]compile-flags:-Zdrop-tracking=no
```

Reply:
Done